src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -938,6 +940,21 @@ data class PlantingSiteObservationSummaryPayload(
       plantingZones = model.plantingZones.map { PlantingZoneObservationSummaryPayload(it) })
 }
 
+data class CompleteAdHocObservationRequestPayload(
+    val conditions: Set<ObservableCondition>,
+    val notes: String?,
+    @Schema(description = "Date and time the observation was performed in the field.")
+    val observedTime: Instant,
+    @Schema(description = "Observation type for this observation.")
+    val observationType: ObservationType,
+    @Schema(description = "The plot name for the ad-hoc plot.") val plotName: String,
```

Comment:
If you put a trailing `//` after the annotation, ktfmt will preserve the line break so the `val` lines up with the other ones. Right now, visually, at first glance it looks like this is an annotation on the `plants` property.